app/api/chemotion/chem_spectra_api.rb

```diff
@@ -196,6 +234,69 @@ def raw_file(att)
         end
       end
 
+      resource :spectra_layouts do
```

Comment:
Cannot use new datatype.
For example, when adding a new NMR type as `Test NMR`, it cannot be used after match.
This is the test file, just change `.txt` to `.jdx`
[test.txt](https://github.com/ComPlat/chemotion_ELN/files/13039673/test.txt)